src/app/modules/submission/submission.utils.ts

```diff
@@ -872,9 +873,15 @@ export const buildMrfMetadata = ({
     submittedSteps ?? [],
     workflow.length,
   )
+
+  const lastSubmittedAt =
```

Comment:
added a lastSubmittedAt for mrf for the responseTimestamp -- checked with Alicia what it should be